### DISCUSSION TITLE
config help

### DISCUSSION THREAD

**Discussion by saep202009** [thuliteio/doks#414] [Q&A]
hi guys pls advice which information i need to setup in AzureCP solution for sharepoint in this fields:

"Application (client) ID" this is ID of what AzureCP or SharePoint App?
"Client Secret" = "XXX" what this values means id ont understand where to find it?

 in example shell are these values
$newAADTenant.ApplicationId = "Application ID" 
$newAADTenant.ApplicationSecret = "XXX"

thanks a lot for answers
Michal

**Comment by saep202009** [comment]
![image](https://user-images.githubusercontent.com/70706181/128726974-fbde5066-6573-4e9e-960b-30974ea85e93.png)

**Comment by h-enk** [comment]
@saep202009, 👉 https://github.com/Yvand/AzureCP 😉
